spark-tpcds-benchmark-runner/var/conf/config.yml

```diff
@@ -33,5 +27,16 @@ spark:
   sparkConf:
     spark.yarn.jars: service/lib/*
     spark.sql.adaptive.enabled: true
-testDataDir: test_data
-iterations: 1
+
+dataScalesGb:
+  - 1
+dataGeneration:
+  parallelism: 5
+  tempWorkingDir: /tmp/spark-benchmark-datagen-working-dir
```

Comment:
might want to make this a subdirectory of `/tmp/spark-benchmark` by default, so that it lines up with the baseUri of the local filesystem